packages/components/inline-adder/src/js/inline-adder.component.js

```diff
@@ -10,6 +10,7 @@ export default {
     onRemove: '&',
     tooltipAdd: '@?',
     tooltipDelete: '@?',
+    rowsLimit: '<?',
```

Comment:
@anooparveti Or did you mean here this line to set the 0 as default value ? (I don't know how to do it, that's why I have the `if (typeof this.rowsLimit === 'undefined') {` which - I can admit - is not very good